src/app/sign-in/page.tsx

```diff
@@ -0,0 +1,81 @@
+export default function LogIn() {
+  return (
+    <main>
+      <div>
+        <div className={styles.parentContainer}>
+          <div className={styles.mainContainer}>
+            <div className={styles.imageContainer}>
+              <img src="eclipse-image.png" className={styles.image} />
+            </div>
+            <div className={styles.formOuterContainer}>
+              <div className={styles.formInnerContainer}>
+                <h1 className={styles.signInText}>SIGN IN</h1>
+                <h3 className={styles.signUpText}>
+                  Don't have an account yet?{" "}
+                  <a href="/signup" className={styles.signUpRefText}>
+                     Sign up!
+                  </a>
+                </h3>
+                <form className={styles.form}>
+                  <div className="w-full">
+                    <label className={styles.label} for="username">
+                      Username
+                    </label>
+                    <input
+                      className={styles.input}
+                      id="username"
+                      type="text"
+                      placeholder="username"
+                    />
+                  </div>
+                  <div className="w-full">
+                    <label className={styles.label} for="password">
+                      Password
+                    </label>
+                    <input
+                      className={styles.input}
+                      id="password"
+                      type="text"
+                      placeholder="password"
+                    />
+                  </div>
+                  <a
+                    href="/forgot-password"
+                    className={styles.forgotPasswordText}
+                  >
+                    Forgot your password?
+                  </a>
+                  <button className={styles.signInButton} type="submit">
+                    Sign In
+                  </button>
+                </form>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+  );
+}
+
+const styles = {
```

Comment:
great attention to detail with the styling, it looks great (especially with no design to go off of!)

one thing - we might want to put color constants in the `tailwind.config.ts` file and then reference their names so we don't have to repeatedly hardcode hex values
[see docs here](https://tailwindcss.com/docs/customizing-colors#using-custom-colors)

Reply:
I agree I think it would also make the styles part easier to read